avm/res/compute/virtual-machine/main.bicep

```diff
@@ -740,19 +740,25 @@ type diagnosticSettingType = {
   @description('Optional. The name of diagnostic setting.')
   name: string?
 
-  @description('Optional. The name of logs that will be streamed. "allLogs" includes all possible logs for the resource. Set to \'\' to disable log collection.')
+  @description('Optional. The name of logs that will be streamed. "allLogs" includes all possible logs for the resource. Set to `[]` to disable log collection.')
```

Comment:
Shouldn't this reflect in a README.md change? I don't see it for vm

Reply:
Will doublecheck

Reply:
ok checked. The VM itself does not support diagnostic settings but the contained NIC deployment does. Will remove it.

Reply:
Fixed / updated only in inner NIC template